CMakeLists.txt

```diff
@@ -214,7 +214,7 @@ endif()
 
 set(DEFAULT_USE_STATIC_LIBATOMIC ON)
 if(UNIX AND NOT APPLE)
-    execute_process(COMMAND cat /etc/os-release COMMAND head -n1 OUTPUT_VARIABLE LINUX_DISTRO)
+    execute_process(COMMAND cat /usr/lib/os-release COMMAND head -n1 OUTPUT_VARIABLE LINUX_DISTRO)

```

Comment:
> Also fixes build not working on arch distros having different names.

Which distribution has this issue?

afak, `/etc/os-release` has a higher priority.

> The file /etc/os-release takes precedence over /usr/lib/os-release. Applications should check for the former, and exclusively use its data if it exists, and only fall back to /usr/lib/os-release if it is missing. 

Link: https://man.archlinux.org/man/os-release.5.en

Reply:
I have CachyOS Linux currently which doesn't return Arch Linux for the previous one. Hence the former change.